tests/TestHarness/CMakeLists.txt

```
@@ -1,4 +1,5 @@
 configure_file(__init__.py . COPYONLY)
+configure_file(core_symbol.py.in core_symbol.py)
```

No need to duplicate the name. Also, in a sea of COPYONLY commands, it might be nice if we indicated this was not accidentally left out. Maybe there's a better comment than I've added?
```suggestion
configure_file(core_symbol.py.in .)  # Updates CORE_SYMBOL_NAME
```